### PR TITLE
SE-2971: Remove inlining from all examples

### PR DESCRIPTION
**Pull request by liberty-askew**
Remove in-lining and replace with writing to property providers for the following examples: 
- check-for-price-outliers 
- check-for-missing-instrument-fields
- relationships

### CODE REVIEW

File: examples/data-qc-checks/check-for-price-outliers/0-upsert-instrument-properties.sql [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+@currentFileLines =
+select Value from Tools.Split
+where Original = @@currentFileContent
+    and Delimiters = '
+' -- newline
+    and [Index] <> 1 -- skip header
+    and Value is not null
+    and Trim(Value, ' ') <> '';
+
+@splitFileLines =
```
Comment (slemasne) [inline examples/data-qc-checks/check-for-price-outliers/0-upsert-instrument-properties.sql:21]: Most of this can be deleted

**Review by slemasne — COMMENTED** [review]
Most of the code in the "upsert properties" file can be deleted.
It is not needed as we are not inlining.

File: examples/data-qc-checks/check-for-price-outliers/0-upsert-instrument-properties.sql (hunk 1 above)
Comment (liberty-askew) [reply examples/data-qc-checks/check-for-price-outliers/0-upsert-instrument-properties.sql]: Have removed it

File: examples/data-qc-checks/check-for-price-outliers/0-upsert-instrument-properties.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
+-- ===============================================================
+-- Description:
+-- In this query, make properties available to Luminesce.
```
Comment (slemasne) [inline examples/data-qc-checks/check-for-price-outliers/0-upsert-instrument-properties.sql:3]: Call file:

`create-instrument-property-definitions`

File: examples/data-qc-checks/check-for-price-outliers/0-upsert-instrument-properties.sql [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+--   we create instrument properties needed for the QC
+-- This approach appends rather than overwrites the current inline property configuration
```
Comment (slemasne) [inline examples/data-qc-checks/check-for-price-outliers/0-upsert-instrument-properties.sql:5]: Update this description

**Review by slemasne — APPROVED** [review]
approved

**Pull request merged**